client/src/features/sessionsV2/components/SessionForm/CustomEnvironmentFields.tsx

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright 2024 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cx from "classnames";
+import { useState, useCallback } from "react";
+import { Controller } from "react-hook-form";
+import { Label, Input, Collapse } from "reactstrap";
+import ChevronFlippedIcon from "../../../../components/icons/ChevronFlippedIcon";
+import { EnvironmentFieldsProps } from "./EnvironmentField";
+import { AdvanceSettingsFields } from "./AdvanceSettingsFields";
+import { SessionLauncherForm } from "../../sessionsV2.types";
+
+export function CustomEnvironmentFields({
+  watch,
+  control,
+  errors,
+}: EnvironmentFieldsProps) {
+  const watchEnvironmentKind = watch("environment_kind");
+  const [isAdvanceSettingOpen, setIsAdvanceSettingsOpen] = useState(false);
+  const toggleIsOpen = useCallback(
+    () =>
+      setIsAdvanceSettingsOpen((isAdvanceSettingOpen) => !isAdvanceSettingOpen),
+    []
+  );
+  return (
+    <div className={cx("d-flex", "flex-column", "gap-3")}>
+      <p className={cx("mb-0")}>
+        Use a custom image to create a session launcher. Provide an image URL,
+        such as one from https://hub.docker.com.
+      </p>
+      <div className={cx("d-flex", "flex-column")}>
+        <Label
+          className={cx("form-label")}
+          for="addSessionLauncherContainerImage"
+        >
+          Container Image
+        </Label>
+        <Controller
+          control={control}
+          name="container_image"
+          render={({ field }) => (
+            <Input
+              className={cx(errors.container_image && "is-invalid")}
+              id="addSessionLauncherContainerImage"
+              placeholder="Docker image"
```

Comment:
We can provide a helper as the placeholder here:

```suggestion
              placeholder="image:tag"
```